File: asp.net/MODEL/ModelBasic.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace MODEL
{
    public abstract class ModelBasic
    {
        private string _modelName;
        public ModelBasic(string modelNameSpace)
        {
            _modelName = modelNameSpace;
        }

        public string getModelNameSpace()
        {
            return _modelName;
        }
    }
}

File: java/j2ee1.6/examples/TestAS4J_PV/src/cn/as4j/model/ModelBasic.java
package cn.as4j.model;


/**
 * Model 基类
 * @author 破晓
 *
 */
public class ModelBasic {

	private String _modelName;
	public ModelBasic(String modelNameSpace)
	{
		_modelName = modelNameSpace;
	}
	
	public String getModelNameSpace()
	{
		return _modelName;
	}
}
